contracts/incentives/ERC20TokenIncentivesController.sol

```diff
@@ -147,6 +146,11 @@ contract ERC20TokenIncentivesController is
     return _usersUnclaimedRewards[_user];
   }
 
+  /// @inheritdoc IAaveIncentivesController
+  function REWARD_TOKEN() external view override returns (address) {
+    return address(TOKEN);
```

Comment:
this cast is not needed. Or maybe you wanted to declare it as `IERC20` on 29th line?